tests/foreman/cli/test_contentcredentials.py

```diff
@@ -102,6 +102,46 @@ def test_positive_get_info_by_name(module_org):
     assert gpg_key['name'] == name
 
 
+@pytest.mark.tier1
+def test_positive_block_delete_key_in_use(module_org):
+    """Create a product and single associated repository. Create a new gpg key and associate
+        it with the product and repository. Attempt to delete the gpg key in use
+
+    :id: 022555fd-e6f2-4c95-80d7-cae26993ca8f
+
+    :expectedresults: deletion of the gpg key is blocked by raising CLIReturnCodeError 70,
+        the key is not deleted, and remains associated with the product and repository
+
+    """
```

Comment:
I think @ColeHiggins2 said this on the last PR, but you can add the BZ here too.